components/molecules/RecommendedRepoCard/recommended-repo-card.tsx

```diff
@@ -3,88 +3,92 @@ import { BiGitPullRequest } from "react-icons/bi";
 import { VscIssues } from "react-icons/vsc";
 import { AiOutlineStar } from "react-icons/ai";
 import { getAvatarByUsername } from "lib/utils/github";
-import StackedAvatar from "../StackedAvatar/stacked-avatar";
+import StackedAvatar, { Contributor } from "../StackedAvatar/stacked-avatar";
+import clsx from "clsx";
+import useFetchRecommendedRepoByRepoName from "lib/hooks/fetchRecommendationByRepo";
+import { truncateString } from "lib/utils/truncate-string";
+import useRepositoryPullRequests from "lib/hooks/api/useRepositoryPullRequests";
+import getPullRequestsContributors from "lib/utils/get-pr-contributors";
+import { Spinner } from "components/atoms/SpinLoader/spin-loader";
 
-export declare interface RecommendedRepoCardProps {
+export declare interface RecommendedRepoCardProps extends React.ComponentProps<"div"> {
   fullname: string;
-  description: string;
-  issues: number;
-  stars: number;
-  pullRequests: number;
-  contributions: DbContribution[];
 }
 
-const RecommendedRepoCard = ({ fullname, description, issues, pullRequests, stars, contributions }: RecommendedRepoCardProps): JSX.Element => {
-
+const RecommendedRepoCard = ({ fullname, className }: RecommendedRepoCardProps): JSX.Element => {
```

Comment:
nit: consistency with most other usage

```suggestion
const RecommendedRepoCard = ({ fullName, className }: RecommendedRepoCardProps): JSX.Element => {
```